authentik/enterprise/api.py

```diff
@@ -29,7 +29,7 @@ class EnterpriseRequiredMixin:
 
     def validate(self, attrs: dict) -> dict:
         """Check that a valid license exists"""
-        if LicenseKey.cached_summary().status != LicenseUsageStatus.UNLICENSED:
+        if LicenseKey.cached_summary().status != LicenseUsageStatus.VALID:
```

Comment:
I don't have any context, but

```suggestion
        if LicenseKey.cached_summary().status.is_valid:
```

?

Reply:
Yep, that looks like a better check